packages/sfui/frameworks/nuxt/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@storefront-ui/nuxt",
-  "version": "1.0.0",
+  "version": "1.0.1-rc.0",
```

Comment:
We need to manually change that to 2.3

Reply:
I dont think it matters since we do not release this package at all

Reply:
Yup, bu we should release it since 2.4 imo

Reply:
true